invoice2data/extract/invoice_template.py

```diff
@@ -31,6 +31,21 @@
 class InvoiceTemplate(OrderedDict):
     """
     Represents single template files that live as .yml files on the disk.
+
+    Methods
+    -------
+    prepare_input(self, extracted_str)
```

Comment:
The `self` argument is passed automatically and shouldn't be included in the docstring.

Reply:
oh, thanks

Reply:
Should I remove it or you are?

Reply:
Feel free to update it yourself. I'm working on Factur-X today.